README.md

```diff
@@ -1,6 +1,6 @@
 # silver-octo-chainsaw
 Practice managed repository
```

Comment:
Might need a better title

Reply:
Like what?

Reply:
"A Tail of Two Birds"

Reply:
That's a great title, Me. I've made the change.